packages/trader/src/App/Components/Elements/TogglePositions/toggle-positions-mobile.jsx

```diff
@@ -22,36 +22,27 @@ const TogglePositionsMobile = observer(
         onClickCancel,
         toggleUnsupportedContractModal,
     }) => {
-        const { symbol, contract_type: trade_contract_type } = useTraderStore();
         const { togglePositionsDrawer, is_positions_drawer_on } = useStore().ui;
         const [hidden_positions_ids, setHiddenPositionsIds] = React.useState([]);
-        const filtered_positions = all_positions
-            .filter(
-                p =>
-                    p.contract_info &&
-                    symbol === p.contract_info.underlying &&
-                    filterByContractType(p.contract_info, trade_contract_type) &&
-                    hidden_positions_ids.every(hidden_position_id => hidden_position_id !== p.contract_info.contract_id)
+        const displayed_positions = filtered_positions
+            .filter(p =>
+                hidden_positions_ids.every(hidden_position_id => hidden_position_id !== p.contract_info.contract_id)
```

Comment:
After Turbos release, we get `filtered_positions` from props, that's why here I only need to filter out (exclude) all positions with `is_sold: 1` that should be hidden. Also, I renamed my `filtered_positions` to `displayed_positions`, and introduced `closed_positions_ids` variable for better readability. The functionality works the same way after this refactoring.